File: async-scala-client/src/main/scala/io/swagger/client/model/Agast_withholding_COFINS.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Agast_withholding_COFINS (
  legalReason: Option[String],
business: Option[Boolean],
businessLegalReason: Option[String],  // reference id to TaxLegalReason
federalGovernment: Option[Boolean],
federalGovernmentLegalReason: Option[String],  // reference id to TaxLegalReason
stateGovernment: Option[Boolean],
stateGovernmentLegalReason: Option[String],  // reference id to TaxLegalReason
cityGovernment: Option[Boolean],
cityGovernmentLegalReason: Option[String]  // reference id to TaxLegalReason
)

File: javascript-closure-angular-client/API/Client/AgastWithholdingCOFINS.js
goog.provide('API.Client.Agast_withholding_COFINS');

/**
 * inform if this item is subject to withholding COFINS on service Transactions, by entity type, default true
 * @record
 */
API.Client.AgastWithholdingCOFINS = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.AgastWithholdingCOFINS.prototype.legalReason;

/**
 * @type {!boolean}
 * @export
 */
API.Client.AgastWithholdingCOFINS.prototype.business;

/**
 * reference id to TaxLegalReason
 * @type {!string}
 * @export
 */
API.Client.AgastWithholdingCOFINS.prototype.businessLegalReason;

/**
 * @type {!boolean}
 * @export
 */
API.Client.AgastWithholdingCOFINS.prototype.federalGovernment;

/**
 * reference id to TaxLegalReason
 * @type {!string}
 * @export
 */
API.Client.AgastWithholdingCOFINS.prototype.federalGovernmentLegalReason;

/**
 * @type {!boolean}
 * @export
 */
API.Client.AgastWithholdingCOFINS.prototype.stateGovernment;

/**
 * reference id to TaxLegalReason
 * @type {!string}
 * @export
 */
API.Client.AgastWithholdingCOFINS.prototype.stateGovernmentLegalReason;

/**
 * @type {!boolean}
 * @export
 */
API.Client.AgastWithholdingCOFINS.prototype.cityGovernment;

/**
 * reference id to TaxLegalReason
 * @type {!string}
 * @export
 */
API.Client.AgastWithholdingCOFINS.prototype.cityGovernmentLegalReason;

